apps/explorer/src/ui/stories/Icons.stories.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import * as Icons from '@mysten/icons';
+import { type Meta } from '@storybook/react';
+
+export default {
+    title: 'Icons',
+} as Meta;
+
+export const RenderIcons = {
```

Comment:
small nit: `Render Icons` in the storybook tree is a little strange - could we up date to `All Icons` or just use `Default` ?